### PR TITLE
Bump csp-adapter

### PR DESCRIPTION
**Pull request by cmurphy**
https://github.com/rancher/rancher/issues/39940

### CODE REVIEW

**Review by cbron — APPROVED** [review]
lgtm, just wait to merge till charts are updated

**Comment by crobby** [comment]
Looks like ci failed due to a missing . in the webhook charts.  Fixed by https://github.com/rancher/webhook/pull/161 and new RC has been cut.

**Comment by cmurphy** [comment]
Thank you!!

File: package/Dockerfile [hunk 1]
```diff
@@ -54,8 +54,8 @@ ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 101.0.0+up0.3.3
 
 # System charts minimal version
 ENV CATTLE_FLEET_MIN_VERSION=101.0.0+up0.5.0
-ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=2.0.0+up0.3.0
-ENV CATTLE_CSP_ADAPTER_MIN_VERSION=2.0.0
+ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=2.0.2+up0.3.2-rc3
+ENV CATTLE_CSP_ADAPTER_MIN_VERSION=2.0.1-rc1
```
Comment (cbron) [inline package/Dockerfile:58]: Webhook should get reverted now, csp-adapter can still move forward

**Review by cbron — APPROVED** [review]

**Pull request merged**